### DISCUSSION TITLE
Saving mission and then updating to a new version of the code (PBO)?

### DISCUSSION THREAD

**Discussion by MadSoloSniper** [Vdauphin/HeartsAndMinds#1089] [Question and answer]
Quick (and hopefully easy) question.  If we "save" the game on one version and then update to the latest version, does that "reset" the mission to the start or do our completed elements become part of the new (updated) version of the game PBO and we just advance from there?

Thanks in advance.

**Comment by Vdauphin** [comment]
Hello,

> If we "save" the game on one version and then update to the latest version, does that "reset" the mission 

No

> do our completed elements become part of the new (updated) version of the game PBO and we just advance from there?

It really depend but as everything is persistant:
so if you change vehicles in your mission.sqm with a game already save, this change will be ignore
If you change the faction, ia already spawned will appear with the previous faction etc

Cheers

**Reply by Vdauphin** [reply]
See also: https://github.com/Vdauphin/HeartsAndMinds/discussions/1005#discussioncomment-690214

**Reply by MadSoloSniper** [reply]
Thank you.  We changed nothing but your Mission PBO to the latest version (no mod changes and no changes to the mission construct) - seems to have kept all of our progress and re-engaged the mission DB just fine where we left off.
Thanks for the quick response,
Solo

**Comment by Raz0rLegend** [comment]
I profiting of this trend to ask a question. 
How does the mission save itself ? I booted a mission yesterday. When I restarted the server, nothing appears again, and all my settings mission disappeared. 
Is it the fact I’m on 4net support server, and not on a dedicated interface ? This is the only reason I saw. I do not understand why if not...

**Reply by GoldJohnKing** [reply]
Login as admin, self-interaction, database, save, wait and done.

**Reply by Vdauphin** [reply]
Yes it is simple like this
If it doesn't work it is because:
- You didn't save
- You fuck up the mission file, download it again from the release page
- The Arma server exe don't have write acces to the file: Vdauphin.vars.Arma3Profile see https://community.bistudio.com/wiki/profileNamespace and https://community.bistudio.com/wiki/Profile

**Reply by Raz0rLegend** [reply]
Yes, i know the way how to. I was speaking how the mission does it. 
So i think it is the 3rd thing. I'm not sure the server is allowed to write inside the A3 profile.
